tests/cases/compiler/reactJsxReactResolvedNodeNextEsm.tsx

```diff
@@ -14,6 +14,7 @@ export const a = <div></div>;
     "main": "",
     "types": "index.d.ts",
     "exports": {
+        ".": "./index.d.ts",

```

Comment:
Like in normal module resolution, a failed `exports` lookup ends the resolution process. We could decide to deviate from normal module resolution and fall back to `types`/`main`/`index.d.ts` if this fails—that would eliminate all the new errors we see in baselines, I think.